src/Stryker.Core/Stryker.Core/Compiling/RollbackProcess.cs

```diff
@@ -94,21 +94,16 @@ public RollbackProcessResult Start(CSharpCompilation compiler, ImmutableArray<Di
 
         private int? ExtractMutationIfAndId(SyntaxNode node)
         {
-            var first = node.GetAnnotations(MutantPlacer.MutationMarkers).FirstOrDefault();
-            if (first == null)
+            var (engine, id) = MutantPlacer.FindEngine(node);
+
+            if (engine == null)
             {
                 return null;
             }
 
-            if (first.Data == null)
-            {
-                Logger.LogDebug("Remove a mutation helper.");
-                return _dummyId;
```

Comment:
If we no longer need the dummyId we should remove the declaration

Reply:
done, sorry about that one